Sources/ShopifyCheckoutKit/CheckoutWebView.swift

```diff
@@ -99,6 +110,13 @@ class CheckoutWebView: WKWebView {
 	func load(checkout url: URL) {
 		load(URLRequest(url: url))
 	}
+
+	private func dispatchPresentedMessage(_ checkoutDidLoad: Bool, _ checkoutDidPresent: Bool) {
+		if !presentedEventDidDispatch && (checkoutDidLoad && checkoutDidPresent) {
+			presentedEventDidDispatch = true
+			CheckoutBridge.sendMessage(self, messageName: "presented", messageBody: nil)
```

Comment:
Could the `sendMessage` here fail? We might want to set the `presentedEventDidDispatch` on the line after the `sendMessge`